### ISSUE TITLE
When you say 3-20 fps, what devices has been tested?

### ISSUE THREAD

**Issue by DuckersMcQuack** [WilliamLCobb/GC4iOS#58]
I got an ipad pro 2018, and curious how mine will fare once a JB hits 12.3.1 and later.

**Comment by c1rca88** [comment]
With iOS 13 we now have controller support!!! Bump for this!

**Comment by DuckersMcQuack** [comment]
YEah, true. But first we need jailbreak support. Apple doesn't allow sideloading, and ios 13 does not have JB, or is even out.

**Comment by tarp2020** [comment]
Is there even an repo to get gc4ios anymore because I've looked on some old posts and I can't really find one. If there is one I'd love to see it.

**Comment by brand175** [comment]
@tarp2020  http://www.willcobb.me/repo/

**Comment by tarp2020** [comment]
Doesn't work for me. I'm on 12.2 chimera the repo won't even load in sileo

**Comment by DuckersMcQuack** [comment]
I just tried it. Windwaker just crashes the app. ios 12.4 on ipad pro 2018 A12X
